ui/components/src/TimeSeriesTooltip/nearby-series.ts

```diff
@@ -151,6 +286,70 @@ export function checkforNearbySeries(
  * If yes, convert from pixel values to logical cartesian coordinates and return all nearby series
  */
```

Comment:
Looks like this was the comment for legacyGetNearbySeries - do we need to change anything about how we describe this newer function?

Reply:
@eunicorn we might not even need the two functions right since xBuffer isn't necessary anymore? Might just leave it for now though just in case and cleanup in a follow up PR when we remove all the LineChart logic

Reply:
i think this function is still called in the line chart tooltip?

but yeah agreed we should just leave it for now until we completely clean up LineChart just to be safe

Reply:
Gotcha, I was thinking that it’s possible the same function can be called in LineChart and TimeChart now, we probably will need to keep LineChart around for a while since it’s used so much internally but yeah, the panel can be simplified a ton once the show_legacy_chart option is removed